FE 2  Rectangular Buttons

dominikapap opened the pull request:
[preview on theme](https://fe-2-buttons-components--melo-envelo.netlify.app/theme)

dominikapap committed:
feat: deafult button component

created default button component in shared/ui folder, used component in app-theme

dominikapap committed:
Merge branch 'develop' into FE-2-buttons-components

dominikapap committed:
feat: added styles

added styles to rect btns and cotainer in theme component

dominikapap committed:
feat: rect buttons

added index file, black and green variatons of btns, changed input from object to variables

dominikapap committed:
fix: singleQuote

changed singleQuote on true

dominikapap committed:
feat: rectangular buttons

removed unnecessary component and folders, moved style for buttons to styles.scss

dominikapap committed:
feat: adjusted styles

adjusted padding and box shadow

dominikapap committed:
refactor: scss styles

removed repeated styles and unnecassary empty lines

akademiaEnvelo commented on src/app/core/theme.component.scss:
```diff
@@ -0,0 +1,19 @@
+.rect-btns-container {
```
nazwa mało mówi, czemu tak to skracamy?

dominikapap replied:
kwestia indywidualnej preferencji :) zgadzam się, że jednak może być mało czytelne, zmieniłam na `rectangular-buttons-container`

LukaszStaniszewski reviewed (approved):
Dla mnie ok.

dominikapap committed:
refactor: styles

changed scss class name to be more readable, moved cursor:pointer to not disabled buttons

akademiaEnvelo commented on src/app/core/theme.component.scss:
```diff
@@ -0,0 +1,19 @@
+  display: flex;
+  margin: 2rem;
+  flex-direction: column;
+  background-color: #fff8;
```
czemu ten kolor nie jest z design systemu?

dominikapap replied:
poprawione, kolor nadany klasą z design systemu

dominikapap committed:
refactor: styles

changed bg color for container, added a class for small button

dominikapap committed:
Merge branch 'develop' into FE-2-buttons-components
